[2024-05-13 13:14:32,940] : INFO: main: welcome....]
[2024-05-17 12:01:58,131] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:01:58,133] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:06:30,424] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:06:30,425] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:07:58,933] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:07:58,935] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:08:32,043] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:08:32,043] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:12:51,920] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:12:51,921] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:18:50,603] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:18:50,603] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:18:50,603] : INFO: common: created directory at: artifacts]
[2024-05-17 12:20:17,807] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:20:17,807] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:20:17,808] : INFO: common: created directory at: artifacts]
[2024-05-17 12:24:58,419] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:24:58,420] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:24:58,421] : INFO: common: created directory at: artifacts]
[2024-05-17 12:25:12,484] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:25:12,484] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:25:12,485] : INFO: common: created directory at: artifacts]
[2024-05-17 12:25:12,486] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-17 12:25:13,177] : INFO: 2146478694: artifacts/data_ingestion/data.zip download with the following information: 
 Connection: close
Content-Length: 2944100
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/octet-stream
ETag: "4450eb63089f56212a41409ae9bcb15b3fb051149cf8d4b895f422d5c86b831c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 40EC:18D16E:86EAC:91B4D:664784E9
Accept-Ranges: bytes
Date: Fri, 17 May 2024 16:25:13 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18149-EWR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1715963114.641583,VS0,VE150
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 2075055866aa6411d1a1e82d0b958788e13ee0e1
Expires: Fri, 17 May 2024 16:30:13 GMT
Source-Age: 0

]
[2024-05-17 12:32:23,056] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:32:23,056] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:32:23,057] : INFO: common: created directory at: artifacts]
[2024-05-17 12:32:23,057] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-17 12:32:23,058] : INFO: 1691874906: file already exists. ~ 2875 KB]
[2024-05-17 12:34:30,686] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:34:30,687] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:34:30,687] : INFO: common: created directory at: artifacts]
[2024-05-17 12:34:30,688] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-17 12:34:31,681] : INFO: 1691874906: artifacts/data_ingestion/data.zip download with the following information: 
 Connection: close
Content-Length: 2944100
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/octet-stream
ETag: "4450eb63089f56212a41409ae9bcb15b3fb051149cf8d4b895f422d5c86b831c"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 40EC:18D16E:86EAC:91B4D:664784E9
Accept-Ranges: bytes
Date: Fri, 17 May 2024 16:34:31 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18120-EWR
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1715963672.856157,VS0,VE103
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 034460a7287f3ef73b64998fc4c85aac8391e55e
Expires: Fri, 17 May 2024 16:39:31 GMT
Source-Age: 0

]
[2024-05-17 12:43:55,781] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:43:55,782] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:43:55,782] : INFO: common: created directory at: artifacts]
[2024-05-17 12:43:55,783] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-17 12:43:56,470] : INFO: 4078411440: artifacts/data_ingestion/data.zip download with the following information: 
 Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: B50C:CDA37:A2F15:B12B4:6647894C
Accept-Ranges: bytes
Date: Fri, 17 May 2024 16:43:56 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18179-EWR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1715964237.792234,VS0,VE128
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: de8d9f0b45e52dfb838488779c5664ec6f4fa755
Expires: Fri, 17 May 2024 16:48:56 GMT
Source-Age: 0

]
[2024-05-17 12:50:42,311] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-17 12:50:42,311] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-17 12:50:42,312] : INFO: common: created directory at: artifacts]
[2024-05-17 12:50:42,313] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-17 12:50:43,711] : INFO: 4078411440: artifacts/data_ingestion/data.zip download with the following information: 
 Connection: close
Content-Length: 4113740
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "654230c3a18a0d8c0894a7fa95c313e8c8804493a71632d5e8b73a38831b1308"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 98DA:8C96B:1ABAD:1E3B6:66478AE3
Accept-Ranges: bytes
Date: Fri, 17 May 2024 16:50:43 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18167-EWR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1715964644.620393,VS0,VE259
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 08252f3baf517bf9e46daac5df70a68e64b3f524
Expires: Fri, 17 May 2024 16:55:43 GMT
Source-Age: 0

]
[2024-05-22 10:30:56,580] : INFO: main: >>>>> Data Ingestion Stage started <<<]
[2024-05-22 10:30:56,584] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-22 10:30:56,587] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-22 10:30:56,587] : INFO: common: created directory at: artifacts]
[2024-05-22 10:30:56,587] : ERROR: main: configManager.get_data_ingestion_config() missing 1 required positional argument: 'self']
Traceback (most recent call last):
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\main.py", line 10, in <module>
    data_ingestion.main()
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\src\text_summarizer\pipeline\stage_01_ingestion.py", line 15, in main
    data_ingestion_config = configManager.get_data_ingestion_config()
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: configManager.get_data_ingestion_config() missing 1 required positional argument: 'self'
[2024-05-22 10:33:54,848] : INFO: main: >>>>> Data Ingestion Stage started <<<]
[2024-05-22 10:33:54,849] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-22 10:33:54,849] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-22 10:33:54,850] : INFO: common: created directory at: artifacts]
[2024-05-22 10:33:54,850] : ERROR: main: configManager.get_data_ingestion_config() missing 1 required positional argument: 'self']
Traceback (most recent call last):
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\main.py", line 10, in <module>
    data_ingestion.main()
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\src\text_summarizer\pipeline\stage_01_ingestion.py", line 14, in main
    data_ingestion_config = configManager.get_data_ingestion_config()
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: configManager.get_data_ingestion_config() missing 1 required positional argument: 'self'
[2024-05-22 10:49:18,725] : INFO: main: >>>>> Data Ingestion Stage started <<<]
[2024-05-22 10:49:18,726] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-22 10:49:18,726] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-22 10:49:18,727] : INFO: common: created directory at: artifacts]
[2024-05-22 10:49:18,727] : ERROR: main: configManager.get_data_ingestion_config() missing 1 required positional argument: 'self']
Traceback (most recent call last):
  File "c:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\main.py", line 10, in <module>
    data_ingestion.main()
  File "c:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\src\text_summarizer\pipeline\stage_01_ingestion.py", line 14, in main
    data_ingestion_config = configManager.get_data_ingestion_config()
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: configManager.get_data_ingestion_config() missing 1 required positional argument: 'self'
[2024-05-22 10:52:05,942] : INFO: main: >>>>> Data Ingestion Stage started <<<]
[2024-05-22 10:52:05,943] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-22 10:52:05,943] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-22 10:52:05,944] : INFO: common: created directory at: artifacts]
[2024-05-22 10:52:05,944] : ERROR: main: 'DataIngestionTrainingPipeline' object has no attribute 'config']
Traceback (most recent call last):
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\main.py", line 10, in <module>
    data_ingestion.main()
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\src\text_summarizer\pipeline\stage_01_ingestion.py", line 14, in main
    data_ingestion_config = configManager.get_data_ingestion_config(self)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\src\text_summarizer\config\configuration.py", line 19, in get_data_ingestion_config
    config = self.config.data_ingestion
             ^^^^^^^^^^^
AttributeError: 'DataIngestionTrainingPipeline' object has no attribute 'config'
[2024-05-22 10:55:30,122] : INFO: main: >>>>> Data Ingestion Stage started <<<]
[2024-05-22 10:55:30,122] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-22 10:55:30,123] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-22 10:55:30,123] : INFO: common: created directory at: artifacts]
[2024-05-22 10:55:30,123] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-22 10:55:31,154] : INFO: data_ingestion: artifacts/data_ingestion/data.zip download with the following information: 
 Connection: close
Content-Length: 4113740
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "654230c3a18a0d8c0894a7fa95c313e8c8804493a71632d5e8b73a38831b1308"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: A8B3:39CFF8:5C52A:656E7:664E0765
Accept-Ranges: bytes
Date: Wed, 22 May 2024 14:55:34 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18167-EWR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1716389734.854189,VS0,VE190
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 345e97d5d85b1dd7f8ebfcc6c124fbda26d92d0a
Expires: Wed, 22 May 2024 15:00:34 GMT
Source-Age: 0

]
[2024-05-22 10:55:31,202] : INFO: main: >>>>> stage Data Ingestion Stage completed <<< 

x===========X]
[2024-05-23 09:49:02,203] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 09:49:02,206] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 09:49:02,208] : INFO: common: created directory at: artifacts]
[2024-05-23 10:05:50,525] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 10:05:50,527] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 10:05:50,528] : INFO: common: created directory at: artifacts]
[2024-05-23 10:14:10,557] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 10:14:10,557] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 10:14:10,558] : INFO: common: created directory at: artifacts]
[2024-05-23 10:14:10,558] : INFO: common: created directory at: artifacts/data_validation]
[2024-05-23 10:25:21,692] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 10:25:21,693] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 10:25:21,693] : INFO: common: created directory at: artifacts]
[2024-05-23 10:25:21,694] : INFO: common: created directory at: artifacts/data_validation]
[2024-05-23 11:00:13,227] : INFO: main: >>>>> Data Ingestion Stage started <<<]
[2024-05-23 11:00:13,228] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 11:00:13,228] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 11:00:13,228] : INFO: common: created directory at: artifacts]
[2024-05-23 11:00:13,228] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 11:00:14,300] : INFO: data_ingestion: artifacts/data_ingestion/data.zip download with the following information: 
 Connection: close
Content-Length: 4113740
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "654230c3a18a0d8c0894a7fa95c313e8c8804493a71632d5e8b73a38831b1308"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 3767:21F7C3:EA4626:10C1093:664F5A01
Accept-Ranges: bytes
Date: Thu, 23 May 2024 15:00:18 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18149-EWR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1716476418.985139,VS0,VE213
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: a00de98770c3232c9505bd6b6481008b002fe70c
Expires: Thu, 23 May 2024 15:05:18 GMT
Source-Age: 0

]
[2024-05-23 11:00:14,347] : INFO: main: >>>>> stage Data Ingestion Stage completed <<< 

x===========X]
[2024-05-23 11:00:14,348] : INFO: main: >>>>> Data Validation Stage started <<<]
[2024-05-23 11:00:14,348] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 11:00:14,349] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 11:00:14,349] : INFO: common: created directory at: artifacts]
[2024-05-23 11:00:14,349] : INFO: common: created directory at: artifacts/data_validation]
[2024-05-23 11:00:14,350] : INFO: main: >>>>> stage Data Validation Stage completed <<< 

x===========X]
[2024-05-23 11:49:37,826] : INFO: config: PyTorch version 2.3.0 available.]
[2024-05-23 11:58:26,067] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 11:58:26,068] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 11:58:26,068] : INFO: common: created directory at: artifacts]
[2024-05-23 11:59:57,132] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 11:59:57,133] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 11:59:57,133] : INFO: common: created directory at: artifacts]
[2024-05-23 12:02:17,450] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 12:02:17,450] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 12:02:17,451] : INFO: common: created directory at: artifacts]
[2024-05-23 12:02:25,771] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 12:02:25,772] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 12:02:25,772] : INFO: common: created directory at: artifacts]
[2024-05-23 12:04:37,683] : INFO: config: PyTorch version 2.3.0 available.]
[2024-05-23 12:04:37,831] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 12:04:37,832] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 12:04:37,833] : INFO: common: created directory at: artifacts]
[2024-05-23 12:08:51,708] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 12:08:51,708] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 12:08:51,709] : INFO: common: created directory at: artifacts]
[2024-05-23 12:10:44,988] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 12:10:44,989] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 12:10:44,989] : INFO: common: created directory at: artifacts]
[2024-05-23 12:14:18,153] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 12:14:18,154] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 12:14:18,154] : INFO: common: created directory at: artifacts]
[2024-05-23 12:22:14,206] : INFO: config: PyTorch version 2.3.0 available.]
[2024-05-23 12:22:19,582] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 12:22:19,582] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 12:22:19,584] : INFO: common: created directory at: artifacts]
[2024-05-23 12:25:17,157] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 12:25:17,157] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 12:25:17,158] : INFO: common: created directory at: artifacts]
[2024-05-23 12:26:36,815] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 12:26:36,816] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 12:26:36,816] : INFO: common: created directory at: artifacts]
[2024-05-23 21:21:23,928] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 21:21:23,931] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 21:21:23,932] : INFO: common: created directory at: artifacts]
[2024-05-23 21:21:23,932] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 21:21:24,946] : INFO: 4078411440: artifacts/data_ingestion/data.zip download with the following information: 
 Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: F52A:C54D1:17BDAE:1AB839:664FEB99
Accept-Ranges: bytes
Date: Fri, 24 May 2024 01:21:29 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18179-EWR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1716513689.233050,VS0,VE137
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 9699061babfb842143df51155b500f74bfdd6eff
Expires: Fri, 24 May 2024 01:26:29 GMT
Source-Age: 0

]
[2024-05-23 21:37:55,424] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 21:37:55,427] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 21:37:55,427] : INFO: common: created directory at: artifacts]
[2024-05-23 21:37:55,428] : INFO: common: created directory at: artifacts/data_validation]
[2024-05-23 21:40:58,743] : INFO: config: PyTorch version 2.3.0 available.]
[2024-05-23 21:41:05,133] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 21:41:05,133] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 21:41:05,135] : INFO: common: created directory at: artifacts]
[2024-05-23 21:55:22,332] : INFO: config: PyTorch version 2.3.0 available.]
[2024-05-23 21:57:00,619] : INFO: config: PyTorch version 2.3.0 available.]
[2024-05-23 21:57:00,757] : INFO: main: >>>>> Data Ingestion Stage started <<<]
[2024-05-23 21:57:00,758] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 21:57:00,759] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 21:57:00,759] : INFO: common: created directory at: artifacts]
[2024-05-23 21:57:00,759] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 21:57:01,446] : INFO: data_ingestion: artifacts/data_ingestion/data.zip download with the following information: 
 Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: F52A:C54D1:17BDAE:1AB839:664FEB99
Accept-Ranges: bytes
Date: Fri, 24 May 2024 01:57:05 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18125-EWR
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1716515826.744916,VS0,VE72
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 509c923baeb210dc78165b0b8a81b2df6beae757
Expires: Fri, 24 May 2024 02:02:05 GMT
Source-Age: 0

]
[2024-05-23 21:57:01,519] : INFO: main: >>>>> stage Data Ingestion Stage completed <<< 

x===========X]
[2024-05-23 21:57:01,520] : INFO: main: >>>>> Data Validation Stage started <<<]
[2024-05-23 21:57:01,521] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 21:57:01,521] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 21:57:01,521] : INFO: common: created directory at: artifacts]
[2024-05-23 21:57:01,522] : INFO: common: created directory at: artifacts/data_validation]
[2024-05-23 21:57:01,522] : ERROR: main: [WinError 3] The system cannot find the path specified: 'artifacts\\data_ingestion\\samsum']
Traceback (most recent call last):
  File "c:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\main.py", line 26, in <module>
    data_validation.main()
  File "c:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\src\text_summarizer\pipeline\stage_02_data_validation.py", line 16, in main
    data_validation.validate_all_files_exist()
  File "c:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\src\text_summarizer\components\data_validation.py", line 30, in validate_all_files_exist
    raise e
  File "c:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\src\text_summarizer\components\data_validation.py", line 15, in validate_all_files_exist
    all_files = os.listdir(os.path.join("artifacts","data_ingestion","samsum"))
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'artifacts\\data_ingestion\\samsum'
[2024-05-23 21:57:49,531] : INFO: config: PyTorch version 2.3.0 available.]
[2024-05-23 21:57:49,676] : INFO: main: >>>>> Data Ingestion Stage started <<<]
[2024-05-23 21:57:49,677] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 21:57:49,677] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 21:57:49,678] : INFO: common: created directory at: artifacts]
[2024-05-23 21:57:49,678] : INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-23 21:57:50,210] : INFO: data_ingestion: artifacts/data_ingestion/data.zip download with the following information: 
 Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: F52A:C54D1:17BDAE:1AB839:664FEB99
Accept-Ranges: bytes
Date: Fri, 24 May 2024 01:57:54 GMT
Via: 1.1 varnish
X-Served-By: cache-ewr18161-EWR
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1716515875.689191,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 4a4ba877a6b3c59bcc78a706c8c0346088b4e8fd
Expires: Fri, 24 May 2024 02:02:54 GMT
Source-Age: 49

]
[2024-05-23 21:57:50,285] : INFO: main: >>>>> stage Data Ingestion Stage completed <<< 

x===========X]
[2024-05-23 21:57:50,285] : INFO: main: >>>>> Data Validation Stage started <<<]
[2024-05-23 21:57:50,287] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 21:57:50,288] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 21:57:50,288] : INFO: common: created directory at: artifacts]
[2024-05-23 21:57:50,288] : INFO: common: created directory at: artifacts/data_validation]
[2024-05-23 21:57:50,289] : INFO: main: >>>>> stage Data Validation Stage completed <<< 

x===========X]
[2024-05-23 21:57:50,290] : INFO: main: >>>>> Data Transformation Stage started <<<]
[2024-05-23 21:57:50,291] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-23 21:57:50,291] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-23 21:57:50,291] : INFO: common: created directory at: artifacts]
[2024-05-23 21:57:52,078] : INFO: main: >>>>> stage Data Transformation Stage completed <<< 

x===========X]
[2024-05-23 22:08:51,670] : INFO: config: PyTorch version 2.3.0 available.]
[2024-05-23 22:48:54,661] : INFO: config: PyTorch version 2.3.0 available.]
[2024-05-29 20:49:27,698] : INFO: server: Started server process [139380]]
[2024-05-29 20:49:27,699] : INFO: on: Waiting for application startup.]
[2024-05-29 20:49:27,699] : INFO: on: Application startup complete.]
[2024-05-29 20:49:27,701] : INFO: server: Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)]
[2024-05-29 20:57:21,024] : INFO: common: yaml file: config\config.yaml loaded successfully.]
[2024-05-29 20:57:21,029] : INFO: common: yaml file: params.yaml loaded successfully.]
[2024-05-29 20:57:21,029] : INFO: common: created directory at: artifacts]
[2024-05-29 20:57:21,029] : INFO: common: created directory at: artifacts/model_evaluation]
[2024-05-29 20:57:21,030] : ERROR: h11_impl: Exception in ASGI application
]
Traceback (most recent call last):
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\transformers\utils\hub.py", line 399, in cached_file
    resolved_file = hf_hub_download(
                    ^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\huggingface_hub\utils\_validators.py", line 106, in _inner_fn
    validate_repo_id(arg_value)
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\huggingface_hub\utils\_validators.py", line 154, in validate_repo_id
    raise HFValidationError(
huggingface_hub.errors.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name': 'artifacts/model_trainer/tokenizer'. Use `repo_type` argument if needed.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\uvicorn\protocols\http\h11_impl.py", line 404, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\uvicorn\middleware\proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\fastapi\applications.py", line 269, in __call__
    await super().__call__(scope, receive, send)
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\starlette\applications.py", line 124, in __call__
    await self.middleware_stack(scope, receive, send)
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\starlette\middleware\errors.py", line 184, in __call__
    raise exc
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\starlette\middleware\errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\starlette\exceptions.py", line 93, in __call__
    raise exc
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\starlette\exceptions.py", line 82, in __call__
    await self.app(scope, receive, sender)
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\fastapi\middleware\asyncexitstack.py", line 21, in __call__
    raise e
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\fastapi\middleware\asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\starlette\routing.py", line 670, in __call__
    await route.handle(scope, receive, send)
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\starlette\routing.py", line 266, in handle
    await self.app(scope, receive, send)
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\starlette\routing.py", line 65, in app
    response = await func(request)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\fastapi\routing.py", line 227, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\fastapi\routing.py", line 160, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\app.py", line 36, in predict
    raise e
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\app.py", line 33, in predict
    text = obj.predict(text)
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\src\text_summarizer\pipeline\prediction.py", line 13, in predict
    tokenizer = AutoTokenizer.from_pretrained(self.config.tokenizer_path)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\transformers\models\auto\tokenization_auto.py", line 817, in from_pretrained
    tokenizer_config = get_tokenizer_config(pretrained_model_name_or_path, **kwargs)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\transformers\models\auto\tokenization_auto.py", line 649, in get_tokenizer_config
    resolved_config_file = cached_file(
                           ^^^^^^^^^^^^
  File "C:\Users\P r x n c E  L i x H\Documents\M_L\NLP\projects\Text_summarizer\textS\Lib\site-packages\transformers\utils\hub.py", line 463, in cached_file
    raise EnvironmentError(
OSError: Incorrect path_or_model_id: 'artifacts/model_trainer/tokenizer'. Please provide either the path to a local folder or the repo_id of a model on the Hub.
[2024-05-29 20:57:48,909] : INFO: server: Shutting down]
[2024-05-29 20:57:49,018] : INFO: on: Waiting for application shutdown.]
[2024-05-29 20:57:49,018] : INFO: on: Application shutdown complete.]
[2024-05-29 20:57:49,018] : INFO: server: Finished server process [139380]]
